### PR TITLE
Devenv

### PR DESCRIPTION
**Pull request by macpijan**
Tested that I could generate reports for Dasharo release binaries from docs.dasharo.org in this environment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,21 +79,29 @@ For more details please refer to the [methodology documentation](docs/methodolog
 
 ## Requirements
 
-- [cbfstool](https://github.com/coreboot/coreboot/tree/master/util/cbfstool)
-  installed on host system
-- [UEFIExtract NE](https://github.com/LongSoft/UEFITool) installed in host
-  system
-- `lzma` compression tool installed on host system
-- `make`
+* Install [Nix pakcage manager](https://nixos.org/download.html)
```
[the remark this answers is not in the document]
Comment (miczyg1) [reply README.md:82]: If we paste the commands anyways, what for do we keep the link? I used the command from the link and failed. When using command wirtten below, it works.

[the remark this answers is not in the document]
Comment (macpijan) [reply README.md:82]: There are two installation methodds (multi-user, and single-user).  I provided command for single-user, as I know that mult-user does not work with-selinux, for instance (and in general, single-user is enough for most cases and easier to remove). How about if I removed the commend and just recommended to use the single-user installation from the link?

[the remark this answers is not in the document]
Comment (macpijan) [reply README.md:82]: I removed the direct commands and provided more description instead.

[the remark this answers is not in the document]
Comment (miczyg1) [reply README.md:82]: Better. Just one more thing:

```suggestion
* Install [Nix package manager](https://nixos.org/download.html) 
```

**Pull request merged**